package.json

```diff
@@ -7,7 +7,7 @@
   "less": "app.less",
   "scripts": {
     "start": "node --preserve-symlinks setup.localhost.js",
-    "postinstall": "node --preserve-symlinks setup.localhost.js compile-less"
+    "compile-less": "node --preserve-symlinks setup.localhost.js compile-less"
```

Comment:
@mattcollier, Is this wrong?

Reply:
Wrong in what way?  Is `npm run compile-less` being executed somewhere now?

Reply:
`compile-less` doesn't exist with the newer views/webpack code.

Reply:
@davidlehn I don't understand what I'm supposed to do here given the information you've put above.

Reply:
I guess I can try to update this to bedrock@3?

Reply:
Got rid of all compile-less/postinstall garbage and upgraded to bedrock@3 in 3f8bc3488c72f9cf0ec23222174b3a32565f4cf5.